pyproject.toml

```
@@ -0,0 +1,64 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name="django-import-export"
```

Noticing inconsistent spacing around the `=` in this, such as:
```suggestion
name = "django-import-export"
```

---

Thanks for reviewing - have corrected

---

You're welcome!  Haven't had a chance to test anything but just happened to see that :). Thanks for all your work on this package!